packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -70,23 +103,46 @@ const bootstrapNewUser = async (
 
   const tms = [] as string[]
   if (isOrganic) {
-    const orgId = generateUID()
-    const teamId = generateUID()
-    tms.push(teamId) // MUTATIVE
-    const validNewTeam = {
-      id: teamId,
-      orgId,
-      name: makeDefaultTeamName(teamId),
-      isOnboardTeam: true
+    const isVerified = identities.some((identity) => identity.isEmailVerified)
+    const teamsWithAutoJoin =
+      isVerified && (await getAutoJoinTeams(usersWithDomainTeams, domain, dataLoader))
```

Comment:
-1 I don't see why we restrict auto join to users without an invite. My mental picture of the auto join feature is like the general channel in Slack, i.e. I would expect every user to initially being added there, even if invited to another team.